client/lxd_instances.go

```diff
@@ -1593,6 +1597,24 @@ func (r *ProtocolLXD) CreateInstanceFile(instanceName string, filePath string, a
 		req.Header.Set("X-LXD-write", args.WriteMode)
 	}
 
+	modifyPerm := []string{}
```

Comment:
you can also just do `var modifyPerm []string` so it'll be initialised to `nil`, you can still append to it and call len on it ok.